### DISCUSSION TITLE
Are there plans to support deletions of images in ghcr?

### DISCUSSION THREAD

**Discussion by badsyntax** [community/community#24974] [Actions]
<p>Similar post but unanswered: <a href="https://github.community/t/deleting-temporary-docker-images-from-ghcr-io-using-actions/138829" class="inline-onebox">Deleting temporary docker images from ghcr.io using actions</a></p>
<p>Just wondering if there are plans or an ETA on providing the ability to delete container images?</p>
<p>I need to create temporary images (to provide app previews on pull requests) and then delete those images when PR’s are closed.</p>

**Comment by clarkbw** [comment]
<p>You can do this already via the UI <a href="https://docs.github.com/en/free-pro-team@latest/packages/guides/deleting-a-container-image" class="inline-onebox">Deleting a container image - GitHub Docs</a></p>
<p>But we’re working on a delete API that could be used on PR merge.  Hoping to have this out by end of month.</p>

**Comment by badsyntax** [comment]
<p>Thanks for the reply! Sorry for not being clear, I was referring to having an API to automate this. Great news to hear one is coming!</p>

**Comment by thomastvedt** [comment]
<p>Hi, do you have an ETA on this?</p>
<p>Also, will <a href="http://ghcr.io" rel="noopener nofollow ugc">ghcr.io</a> packages be exposed in Github GraphQL API? I need to be able to query for a list of packages matching a search string :slight_smile:</p>

**Comment by clarkbw** [comment]
<p>Rolled out yesterday. <a href="https://github.community/t/you-can-now-delete-and-restore-any-package-type-within-github-packages/162393" class="inline-onebox">You can now delete and restore any package type within GitHub Packages</a></p>

**Reply by felipecrs** [reply]
@clarkbw, I was trying to delete an image with skopeo today (https://github.com/containers/skopeo/issues/1880#issuecomment-1544334859), and I received a "The operation is unsupported" error.

The link you refers to seems to be dead. Would you be able to provide more information about it?

**Reply by clarkbw** [reply]
https://github.com/orgs/community/discussions/26267 and https://github.blog/changelog/2021-02-17-packages-delete-and-restore-for-all-packages/

**Comment by bougyman** [comment]
Has there been any solution here? The "answer"  goes to a dead link. Still get 
```
{"errors":[{"code":"UNSUPPORTED","message":"The operation is unsupported."}]}
 (405 Method Not Allowed)
 ```

When trying to delete an image.

**Comment by adriens** [comment]
Hi guys, any news about this issue ?
